br/pkg/lightning/backend/external/writer.go

```diff
@@ -470,23 +439,9 @@ func (w *Writer) flushKVs(ctx context.Context, fromClose bool) (err error) {
 	}()
 
 	sortStart := time.Now()
-	if w.shareMu != nil {
-		sorty.MaxGor = min(8, uint64(variable.GetDDLReorgWorkerCounter()))
```

Comment:
we don't need `sorty` library? It has better performance

Reply:
Do you mean setting `sorty.MaxGor` to `1`?

Reply:
fd9caaa233a